srcpkgs/wxMaxima/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'wxMaxima'
 pkgname=wxMaxima
-version=16.12.2
-revision=5
-wrksrc="wxmaxima-${version}"
-build_style=gnu-configure
-configure_args="--with-wx-config=wx-config-3.0 --enable-printing"
-makedepends="wxWidgets-devel"
+version=18.02.0
+revision=1
+wrksrc="wxmaxima-Version-${version}"
+build_style=cmake
+configure_args="-DwxWidgets_CONFIG_EXECUTABLE=/usr/bin/wx-config-gtk3"
+makedepends="wxWidgets-gtk3-devel"
 depends="maxima desktop-file-utils"
 short_desc="Cross-platform GUI for the computer algebra system Maxima"
 maintainer="Martin Riese <grauehaare@gmx.de>"
-homepage="https://andrejv.github.io/wxmaxima/"
 license="GPL-2"
```

Comment:
Still no check by xlint?

Reply:
No we don't check on GPL-2 and other invalid SPDX identifiers we didn't even work out how we will do it and even store valid SPDX identifiers .